zScanner/DomainModel/DepartmentDomainModel.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  DepartmentDomainModel.swift
+//  zScanner
+//
+//  Created by Jan Provazník on 14/02/2020.
+//  Copyright © 2020 Institut klinické a experimentální medicíny. All rights reserved.
+//
+
+import Foundation
+
+struct DepartmentDomainModel {
+    var id: String
+    var name: String
+}
+
+extension DepartmentDomainModel: Equatable {
+    static func == (lhs: DepartmentDomainModel, rhs: DepartmentDomainModel) -> Bool {
+        return lhs.id == rhs.id
+    }
+}
```

Comment:
You don't need to implement the `==` function. Just add Equatable directly to the first definition.. 
Or is there a reason why you only compare id instead of id and name?  

Where are you comparing it?

Reply:
There is actually no reason to compare only by id. Thank you